src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -431,9 +431,9 @@ internal enum EnabledValue
             ExplicitOptOut
         }
 
-        public static EnabledValue ParseEnableValue(string value)
+        public static EnabledValue ParseEnableValue(string? value)
         {
-            if (string.Equals(value, "default", StringComparison.OrdinalIgnoreCase))
+            if (string.IsNullOrEmpty(value))
```

Comment:
Remind me what happens if I set the value to something like "Foo"?  This returns `EnabledValue.Undefined` and the functionality is disabled?

Reply:
Yes, currently the invalid value would disable audit.

This reminds me I wasn't sure what we should should when the value is `Foo` and I wanted to ask others opinion if we should default to implicit opt in, and output a message, similar to when we parse `NuGetAuditMode`?

https://github.com/NuGet/NuGet.Client/blob/3dbfac4cf9633d012ff22536cdb895e2f4e4b487/src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs#L419-L423

Reply:
IMO false should disable or opt-out of the feature. Any other value should be the same as the default behavior which is "enabled". Accordingly, it'd be great if we could simplify the logic as suggested in the other comment.

Reply:
I changed `ParseEnableValue` to use the same pattern as `ParseAuditMode` and `ParseAuditLevel`.